content/blog/2025/05/2025-05-09-twim.md

```diff
@@ -0,0 +1,170 @@
++++
+date = "2025-05-09T17:00:00Z"
+title = "This Week in Matrix 2025-05-09"
+path = "/blog/2025/05/09/this-week-in-matrix-2025-05-09"
+
+[taxonomies]
+author = ["MTRNord"]
+category = ["This Week in Matrix"]
++++
+
+## Matrix Live
+
+{{youtube_player(video_id="0dRWKVSAh1U")}}
+
+## Dept of elections 🗳️
+
+[Robin Riley (m.org)](https://matrix.to/#/@robinriley:matrix.org) reports
+
+> It's [Governing Board](https://matrix.org/foundation/governing-board/) election season and nominations opened last weekend! This year we're electing representatives for Gold, Silver, Individual, and Associate Members. Learn more on the [2025 election page](https://matrix.org/foundation/governing-board-elections/2025/) and consider nominating yourself! You can also drop by the [Office of the Governing Board](https://matrix.to/#/#governing-board-office:matrix.org) to talk to current representatives and community members.
+
+## Dept of Servers 🏢
+
+### Matrix Authentication Service (MAS) 0.16.0 ([website](https://github.com/element-hq/matrix-authentication-service/))
+
+MAS (Matrix Authentication Service) is an OAuth 2.0 and OpenID Provider server for Matrix.
```

Comment:
I [suggest](https://github.com/matrix-org/twim-config/issues/43)
```suggestion
Matrix Authentication Service (MAS) is an OAuth 2.0 and OpenID Provider server developed by Element.
```
to match synapse